homeassistant/components/habitica/strings.json

```
@@ -25,7 +25,13 @@
     "tag_options_name": "Tags",
     "tag_options_description": "Add or remove tags from a task.",
     "name_description": "The title for the Habitica task.",
-    "cost_name": "Cost"
+    "cost_name": "Cost",
+    "difficulty_name": "Difficulty",
+    "difficulty_description": "Sets the task difficulty.",
```

```suggestion
    "difficulty_description": "The difficulty of the task.",
```
to match the other descriptions here